### ISSUE TITLE
Add git revert content

### ISSUE THREAD

**Issue by Fibii** [JackLot/LearnGit.io#42]
I need to use it, wanted to check your content, didn't find it

**Comment by JackLot** [comment]
Good call. I do cover over `reset` and `--amend` but not `revert`. Probably a good lesson to add the the utility command learning track
